extensions/amp-analytics/amp-analytics.md

```diff
@@ -137,56 +138,85 @@ The `vars` attribute in the configuration can be used to define new key-value pa
 }
 ```
 
-### Triggers
+##### Triggers
 The `triggers` attribute describes when an analytics request should be sent. It contains a key-value pair of trigger-name and
  trigger-configuration. Trigger name can be any string comprised of alphanumeric characters (a-zA-Z0-9). Triggers from a
  configuration with lower precedence are overridden by triggers with the same names from a configuration with higher precedence.
 
-  - `on` (required) The event to listener for. Valid values are `visible`,`click`, and `timer`.
+  - `on` (required) The event to listener for. Valid values are `click`, `scroll`, `timer`, and `visible`.
   - `request` (required) Name of the request to send (as specified in the `requests` section).
   - `vars` An object containing key-value pairs used to override `vars` defined in the top level config, or to specify
     vars unique to this trigger.
+  - `selector` (required when `on` is set to `click`) This configuration is used on conjunction with the `click` trigger. Please see below for details.
```

Comment:
At that point, we can rework it. The main point here is to give visibility into all of the possible children of `triggers` and the situations in which they are required/optional/irrelevant.